images/module-dependencies.dot

```diff
@@ -0,0 +1,8 @@
+digraph {
+  Update -> Main
+  View -> Main
+  Model -> Main
+  Model -> Update
+  Model -> View
+  Flags -> Main
+}
```

Comment:
I originally set a default edge attribute:

```dot
digraph {
  edge [dir=back]

  Update -> Main
  View -> Main
  ...
}
```

But, maybe that's not the most robust way...